### ISSUE TITLE
Selenium Failure Report

### ISSUE THREAD

**Issue by TomatoesBurner** [TomatoesBurner/Agile-Web-Community-Forum#65]
Apologies for not finishing testing the system with the Selenium framework before submitting it on the 19th.In order not to crash the project as a whole, I didn't push my local system testing code to github.
So far I've caused the following major bugs, so let's be warned.
1. In Selenium's test function **setup** uses **'print("Database URI: ", cls.app.config['SQLALCHEMY_DATABASE_URI'])',** which prints out the correct test memory database address: memory, but the data is still injected into our development database     **SQLALCHEMY_DATABASE_URI = os.environ.get('DATABASE_URL') or \   'sqlite:///' + os.path.join(BASE_DIR, 'app.db')**. I chose to discard the memory database and use the same type of hard drive database as app.db, testapp.db. The data is still not located correctly.
2. **tearDown** method can not be correctly deleted data, that is, in the case of the existence of 1, tearDown can not be pulled out of the test data, after I printed the log found that tearDown to look for the database address is still the test database, rather than the development of the database, resulting in the failure of data deletion.
In summary, in order not to affect our display and development of the database used, and in tutor to us to correct the project caused by the programme error, I decided to delete this piece of code.

Causes (guesses).
1. Multi-threaded processing caused by the test class start time, the creation of multiple test objects .

I still have not found a solution, again and members of the group to apologise.
